libs/components/datetime/src/lib/modules/timepicker/timepicker.component.ts

```diff
@@ -175,94 +173,106 @@ export class SkyTimepickerComponent implements OnInit, OnDestroy {
         });
       });
     }
   }
 
-  private get timepickerRef(): ElementRef {
-    return this._timepickerRef;
+  public get timepickerRef(): ElementRef | undefined {
+    return this.#_timepickerRef;
   }
 
   @ViewChild('timepickerTemplateRef', {
     read: TemplateRef,
   })
-  private timepickerTemplateRef: TemplateRef<unknown>;
+  public timepickerTemplateRef: TemplateRef<unknown> | undefined;
 
   @ViewChild('triggerButtonRef', {
     read: ElementRef,
   })
-  private triggerButtonRef: ElementRef;
+  public triggerButtonRef: ElementRef | undefined;
 
   @ViewChild('inputTemplateRef', {
     read: TemplateRef,
     static: true,
   })
-  private inputTemplateRef: TemplateRef<unknown>;
+  public inputTemplateRef: TemplateRef<unknown> | undefined;
 
   @ViewChild('triggerButtonTemplateRef', {
     read: TemplateRef,
     static: true,
   })
-  private triggerButtonTemplateRef: TemplateRef<unknown>;
+  public triggerButtonTemplateRef: TemplateRef<unknown> | undefined;
+
+  #affixer: SkyAffixer | undefined;
 
-  private affixer: SkyAffixer;
+  #timepickerUnsubscribe = new Subject<void>();
 
-  private timepickerUnsubscribe: Subject<void>;
+  #ngUnsubscribe = new Subject<void>();
 
-  private ngUnsubscribe = new Subject<void>();
+  #overlay: SkyOverlayInstance | undefined;
 
-  private overlay: SkyOverlayInstance;
+  #overlayKeydownListner: Subscription | undefined;
 
-  private overlayKeydownListner: Subscription;
+  #_disabled = false;
 
-  private _disabled: boolean;
+  #_timepickerRef: ElementRef | undefined;
 
-  private _timepickerRef: ElementRef;
+  #affixService: SkyAffixService;
+  #changeDetector: ChangeDetectorRef;
+  #coreAdapter: SkyCoreAdapterService;
+  #overlayService: SkyOverlayService;
 
   constructor(
-    private affixService: SkyAffixService,
-    private changeDetector: ChangeDetectorRef,
-    private coreAdapter: SkyCoreAdapterService,
-    private overlayService: SkyOverlayService,
+    affixService: SkyAffixService,
+    changeDetector: ChangeDetectorRef,
+    coreAdapter: SkyCoreAdapterService,
+    overlayService: SkyOverlayService,
     @Optional() public inputBoxHostService?: SkyInputBoxHostService,
     @Optional() themeSvc?: SkyThemeService
   ) {
+    this.#affixService = affixService;
+    this.#changeDetector = changeDetector;
+    this.#coreAdapter = coreAdapter;
+    this.#overlayService = overlayService;
+
     const uniqueId = nextId++;
     this.timepickerId = `sky-timepicker-${uniqueId}`;
     this.triggerButtonId = `sky-timepicker-button-${uniqueId}`;
 
     // Update icons when theme changes.
     themeSvc?.settingsChange
-      .pipe(takeUntil(this.ngUnsubscribe))
+      .pipe(takeUntil(this.#ngUnsubscribe))
       .subscribe(() => {
-        this.changeDetector.markForCheck();
+        this.#changeDetector.markForCheck();
       });
   }
 
   public ngOnInit(): void {
     this.setFormat(this.timeFormat);
-    this.addKeydownListner();
+    this.#addKeydownListner();
```

Comment:
While we're changing the name of this property, we can also fix its spelling. `addKeydownListner` 😎